content/blog/2019/05/2019-05-09-chinese-localization.adoc

```diff
@@ -0,0 +1,44 @@
+---
+:layout: post
+:title: A Big Step of the Chinese Localization
+:tags:
+- general
+- community
+- localization
+- chinese
+:author: linuxsuren
+---
+
+Since 2017, I started to do some contributions to the Jenkins community. As a beginner, translation might be
+the easiest way to involve. You don't need to understand the whole context, even to create a ticket in the
+issue tracker system. Improvement of localization usually is minor. But some problems occurred soon,
+there isn't a native speaker of Chinese could help to do the real review for my PRs. So, sometimes my PRs
+are hard to be merged into master.
+
+Some contributors told me that I can start a thread at the mailing list. Normally, discussing at the mailing list
+is the open source community way. We got a lot of ideas for the localization from there. As a result,
+we achieved some goals that I'd like to share here.
+
+== JEP-216
+
+Previously all language files were aggregated in one place for core and for each plugin.
+For https://github.com/jenkinsci/jep/blob/master/jep/216/README.adoc[this proposal], each language has a single localization plugin, such as https://github.com/jenkinsci/localization-zh-cn-plugin[Chinese Localization plugin].
+Finally, https://github.com/jenkinsci/localization-support-plugin[Localization Support Plugin] and
+https://github.com/jenkinsci/localization-zh-cn-plugin[Chinese Localization plugin] are able to support
+all kinds of existed i10n resource files. From the https://plugins.jenkins.io/localization-zh-cn[plugins website],
+you can see that the installs number already is 13k.
```

Comment:
the installs number already is 13k
-->
the number of installation is already 13k
?

Reply:
Fixed with another way.